Frigate Camera Notifications/Beta

```diff
@@ -1072,7 +1081,7 @@ action:
                           title: "{{title}}"
                           message: "{{message}}"
                           data:
-                            tag: "{{ id }}{{'-loitering' if loitering}}"
+                            tag: "{{ id }}"
```

Comment:
is there a reson this was changed. I personally have never enabled loitering but if there is something i should know please let me know.

Reply:
This was on the "new" event notification. It was inconsistent with the other two notification types in the section, and I don't believe you can get a new event that is also loitering, anyways. It would have to be an update, since loitering is some amount of time since the event began, right?